docs/static_site/src/_includes/get_started/macos/perl/perl.md

```diff
@@ -1 +1 @@
-Refer to the [Perl section of installation guide](/get_started/osx_setup.html#install-the-mxnet-package-for-perl).
\ No newline at end of file
+Refer to the [Perl section of the MXNet macOS installation guide](/get_started/osx_setup.html#install-the-mxnet-package-for-perl).
```

Comment:
MacOS?

Reply:
I was confused which to use too. In wiki it's macOS.